modules/search/instant-search/components/overlay.jsx

```diff
@@ -11,23 +11,23 @@ const closeOnEscapeKey = callback => event => {
 	event.key === 'Escape' && callback();
 };
 
-const Overlay = ( { showOverlay, toggleOverlay, children } ) => {
+const Overlay = ( { shouldShowOverlay, closeOverlay, children } ) => {
 	useEffect( () => {
-		window.addEventListener( 'keydown', closeOnEscapeKey( toggleOverlay ) );
+		window.addEventListener( 'keydown', closeOnEscapeKey( closeOverlay ) );
 		return () => {
 			// Cleanup after event
-			window.removeEventListener( 'keydown', closeOnEscapeKey( toggleOverlay ) );
+			window.removeEventListener( 'keydown', closeOnEscapeKey( closeOverlay ) );
 		};
 	}, [] );
 
 	const classNames = [ 'jetpack-instant-search__overlay' ];
-	if ( ! showOverlay ) {
+	if ( ! shouldShowOverlay ) {
```

Comment:
Why the distinction from `showOverlay` to `shouldShowOverlay`?

Reply:
We had two props for Overlay: `showOverlay` and `closeOverlay` which made them sound like they were both functions that took the described actions. I wanted to make it clear that `showOverlay` was a boolean. Is there a better name? `isOverlayActive`?

Reply:
Ah, gotcha. `shouldShowOverlay` makes sense, then. `isVisible` might also work as well.

Reply:
`isVisible` is clearer! Thanks. I'll change it.